src/store/modules/draft-message/index.ts

```diff
@@ -31,11 +31,17 @@ const mutations: MutationTree<DraftState> = {
   },
 
   deleteReplyTold(state, payload: { partnerId: string; replyToId: string }) {
-    delete state.drafts[payload.partnerId].replyToId
+    const draft = state.drafts[payload.partnerId]
+    if (draft) {
+      delete draft.replyToId
+    }
   },
 
   deleteMessage(state, payload: { partnerId: string; message: string }) {
-    delete state.drafts[payload.partnerId].message
+    const draft = state.drafts[payload.partnerId]
+    if (draft) {
+      delete draft.message
+    }
```

Comment:
Is this related to the current task? Could you describe the bug?

Reply:
Bug description:
After some time, if you return to the PWA screen. Then the selected command is not sent when you press Enter. Because when we send a message, we try to remove it from the draft, but the draft is empty at the moment.